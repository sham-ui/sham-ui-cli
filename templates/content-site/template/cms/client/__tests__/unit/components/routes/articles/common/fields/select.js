// eslint-disable-next-line max-len
import RoutesArticlesCommonFieldsSelect  from '../../../../../../../src/components/routes/articles/common/fields/select.sfc';
import { oninput } from 'sham-ui-directives';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( RoutesArticlesCommonFieldsSelect, {
        directives: {
            oninput
        }
    } );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
