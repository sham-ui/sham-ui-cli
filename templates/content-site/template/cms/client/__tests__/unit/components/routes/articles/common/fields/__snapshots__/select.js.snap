// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Options": Object {
    "ID": "component",
    "directives": Object {
      "oninput": [Function],
    },
    "emptyItem": Object {
      "id": null,
      "name": "- - -",
    },
    "items": Array [],
    "onChange": [Function],
    "value": null,
  },
  "html": "
<select>
  <option value=\\"null\\">- - -</option>
</select>
",
}
`;
